.phpstan-baseline.php

```diff
@@ -1555,6 +1555,18 @@
 	'count' => 1,
 	'path' => __DIR__ . '/src/Glpi/Api/API.php',
 ];
+$ignoreErrors[] = [
```

Comment:
Messages from deprecated API can be ignored inline, no?

Reply:
As it shares the same Symfony controller with the new API, I think the code should be refactored a bit to return a Symfony Response.

Reply:
OK, no problem